File extension .eot not supported

## Description and Context

I got a weird problem in my HS project: .eot font files didn't get uploaded to my theme, throwing this error:

```
[ERROR] The file "sr-theme-customized/fonts/suisse-intl/SuisseIntl-Black-WebS.eot" does not have a valid extension
```

### Steps to reproduce

Upload a file with the .eot extension using `hs upload`.

### What I tried

I made a fork and tried to add `eot` to the ALLOWED_EXTENSIONS list, see here https://github.com/th0rgall/hubspot-cms-tools/pull/1/files#diff-c360fc5df10a4024db0e2534d8de43092761d7a1e2805bac489a5a76e25fdc19R20

I assume this "passes" the eot file through the client-side CLI validations for files. But when I tested this change, I got a different error

```
[ERROR] Failed to process file /<omitted>/sr-theme-customized/fonts/suisse-intl/SuisseIntl-Black-WebS.eot. File extension 'eot' is not supported
```

Since I didn't see any other references to allow lists in the CLI client code, I assume this is a HubSpot server validation error, which I'm not able to fix.

## Expected behavior

Allow fonts with the .eot extension to be uploaded.

--- a/packages/cli-lib/lib/constants.js
+++ b/packages/cli-lib/lib/constants.js
@@ -16,6 +16,7 @@ const ALLOWED_EXTENSIONS = new Set([
   'gif',
   'map',
   'svg',
+  'eot',
   'ttf',
   'woff',
   'woff2',

